src/subprocess-posix.cc

```diff
@@ -123,6 +127,12 @@ bool Subprocess::Start(SubprocessSet* set, const string& command) {
   if (err != 0)
     Fatal("posix_spawn: %s", strerror(err));
 
+  if(priority != INT_MIN) {
```

Comment:
why does this check exist? It looks like the only values passed by your code are 0 and 1.